tests/integration/security/main_test.go

```diff
@@ -60,21 +59,6 @@ meshConfig:
     gatewayTopology:
       numTrustedProxies: 1 # Needed for X-Forwarded-For (See https://istio.io/latest/docs/ops/configuration/traffic-management/network-topologies/)
 `
-			} else {
-				cfg.ControlPlaneValues = `
-values:
-  pilot: 
-    env: 
-      PILOT_JWT_ENABLE_REMOTE_JWKS: true
```

Comment:
I think we are losing this and L74 that we still want

Reply:
interesting the tests pass...

Reply:
It was already there in the if part too.. I just removed the if else..